package/tests/test_repos_codeberg.py

```diff
@@ -15,12 +11,18 @@ class MockClient(ForgejoClient):
     participants and archiving packages.
     """
 
+    def __init__(self):
+        # Avoid calling the parent constructor which requires a base_url argument
+        self.base_url = "https://codeberg.org"
+        self.api_base_url = f"{self.base_url}/api/v1"
```

Comment:
No need to avoid calling the parent `__init__` method, you could just pass in the `base_url` to it and avoid the small duplication:

```suggestion
    def __init__(self):
        super().__init__(base_url="https://codeberg.org")
```